src/scss/root/_index.scss

```diff
@@ -0,0 +1,112 @@
+@use 'sass:list';
+@use 'sass:map';
+
+@forward 'reset';
```

Comment:
Shouldn't this be `@use`? We just want the contents output - consumers don't need access to anything in these files

Reply:
changed, tested, not needed to be forwarded